src/SprykerEco/Yves/Ratepay/Theme/default/partial/ratepay-summary-widget.twig

```diff
@@ -0,0 +1,24 @@
+{% extends template('widget') %}
+
+{% define data = {
+    ratepayPaymentTransfer: _widget.ratepayPaymentTransfer,
+    quoteTransfer: _widget.quoteTransfer,
+    view: _widget.view
+} %}
+
+{% block body %}
+    {% if data.ratepayPaymentTransfer is not null %}
+        {% include '@Ratepay/partial/summary/' ~ data.view ~ '.twig' with {
```

Comment:
We are not using partials. You have to create a view for every partial file you have and extend template.